docs/spelling_wordlist.txt

```diff
@@ -122,6 +122,7 @@ avp
 Avro
 avro
 aws
+AwsBaseAsyncHook
```

Comment:
Ah, this will be needed when #36876 is merged (soon i hope), i cherry picked it for testing and got some spelling errors there when building docs.

Reply:
That is odd
I don't think the right fix is to add keywords that you didn't add to the exclusion list.

I wonder if maybe this automated process adds new words to the doc build (which were not there before) thus it means that everytime we make a release we will need to handle more new words to the exclusion list?

Reply:
In the doc page we are providing instructions on what a user should use instead of a deprecated entity, so it's usually a message with path or name of another Airflow class,method, function etc. So i think yes, it may happen that with every build there might be a new "unknown" word if there is a new deprecation and new class included in the deprecation message. I am not sure if there is a way to avoid it? Developers can put anything in the deprecation message and it may cause spelling checks to fail, but it should be caught in the PR that adds the deprecation.

Reply:
@kacpermuda -> in case of those things that were added, we usually do it differently - names of the classes are usually `BaseAsyncSessionFactory` quoted - to make their fonts fixed width, so that they look like a code. And that automatically disables spell-checking for those names. That's why our spellchecking wordlis is not cluttered with 100s of those.

Reply:
@potiuk That makes sense, I will adjust the class names that were missing backticks, so that i don't have to whitelist them in spellchecking. 

That leaves me with the question: if we start including deprecation message in the docs, when a developer forgets to put those backticks in the deprecation message, the spellcheck will fail and it will be to the person accepting the PR to make sure that they do not add those words to the whitelist but put backticks around them? Is there any way to add a clear information to the developer about why the spellcheck failed in this scenario so that he can correct himself ?

Reply:
Sure - we have docs, we can explain there, also there are messages printed after spellcheck fail, we already print quite a lot of information there - for example how you can build docs faster with selecting individual packages. Maybe a good idea to open PR with such change. 

Docs Troubleshooting:
  
https://github.com/apache/airflow/blob/main/docs/README.rst#docs-troubleshooting 


Hints (we print them coloured, to make them stand out): 

 https://github.com/apache/airflow/blob/main/docs/build_docs.py#L81
 https://github.com/apache/airflow/blob/9f4f208b5da38bc2e82db682c636ec4fcf7ad617/docs/build_docs.py#L554  

You could check if there are spelling errors and give some hints to the user if they do. That would be a nice improvement  - probably you are the right person to describe it in the way that will help others like you, who had this situation :)

Reply:
Nice!